### ISSUE TITLE
Build Novel Project pane needs a draggable divider

### ISSUE THREAD

**Issue by johnblommers** [vkbo/novelWriter#247]
In the Build Novel Project pane there's a Novel Title Formats section to enter Title, Chapter, Unnumbered, Scene and Section. In my case the Chapter and Scene formats are too long to display in their respective fields:

```
Chapter: Chapter %chnumword% \\ %title%
Scene:   Scene %chnum%.%scnum% \\%title%
```

So I naturally moved the cursor to the boundary between the left hand pane and the preview pane and ... there is no divider there to drag on! 

Would you consider adding a draggable divider?

**Comment by vkbo** [comment]
Those are QLineEdit boxes, and for some reason they don't by default expand like other boxes do. I've never figured out whether that is a bug or some obscure setting somewhere, but I have up until now just given them a fixed size. I can increase it by a bit perhaps. It's currently 200px, 250px seems to accommodate your formats, depending on your UI font size.

**Comment by vkbo** [comment]
I've played around with it, and there definitely is some issue with QLineEdit not wanting to expand in a QGridLayout. It works just fine in other layout containers. I wrapped them all in a horizonatl box, and they expand just fine.

However, this entire approach is very hacky, and adds a number of dummy boxes to the code, and I'm not sure it is a good solution. I think I will stick to setting a slightly larger fixed size instead.

One thing I can do, and kindof want to anyway, is rename two of the longest replace keywords.
* `%chnumword%` to `%chword%`
* `%scabsnum%` to `%scabs%`

It makes it a bit less verbose.
